scripts/upgrade.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+# Copyright (c) HashiCorp, Inc.
+# SPDX-License-Identifier: MPL-2.0
+
+set -ex
+
+PROJECT_ROOT=$(cd "$(dirname "${BASH_SOURCE:-$0}")/.." && pwd)
+CDKTF_VERSION=$1
+
+if [ -z "$CDKTF_VERSION" ]; then
+  echo "Usage: $0 <cdktf-version> <kubernetes-prebuilt-version>"
```

Comment:
```suggestion
  echo "Usage: $0 <cdktf-version>"
```
Or am I missing something?